doc/part-1-github-ui/workflows-and-best-practices.md

```diff
@@ -26,6 +26,15 @@ Workflow common practices and best practices
 
 ## Tasks
 
+You will now learn to use everything you have learnt about in the learning videos. You should not have any issues while working on this task, if you follow the instructions to the latter. But incase you run into any issues, feel free to ask for help in the [Github discussion for part 1.](https://github.com/Ifycode/git-github-training/discussions/79) Or you can send the pull request anyway, and state the issue you have there in your pull request.
+
+**Task**: Follow the instructions in the issue created from the template in the tasks section of the [github contributions page.](https://obiagba-mary.gitbook.io/git-and-github-training/part-1-github-ui/github-contributions). This time, the you are to interact with the develop branch as shown in the learning videos.
+
+{% hint style="info" %}
+Note
+Interested contributors should read up How to submit your changes to this training resource for guidelines on [how to make contributions to this repository](https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/making-and-submitting-changes)
+{% endhint %}
+
```

Comment:
Madam, what is contributors-related hint doing inside tasks section? 😂

Change the the hint to the correct hint. Again, see the related issue to know what hint to use.
You don't have to do it tonight. Rest if you need to. You will be able to think clearly and see your mistake 👍